### ISSUE TITLE
 How to evaluate scared dataset and servct dataset?

### ISSUE THREAD

**Issue by Tobyzai** [wurenkai/MCF-SMSIS#2]
Hello.  Thanks for your work. 
How to evaluate scared dataset and servct dataset? I am using the pre-training model S3-d1sd2 to test 16 pairs of images from the servct dataset, with the image size padded to (736, 576) by right padding. The results are saved using test_save_disp.py and cropped back to the original image size (720, 576). But got poor evaluation parameters, epe:35.30,bad3:99.89.

May I ask why this is?
If possible, please give the complete evaluation code for scared dataset and servct dataset.

**Comment by wurenkai** [comment]
Hi, thanks for your interest in our work. Based on your question, I realized that you are in need of testing on SERV-CT dataset. However, the weights given in our project are for SCARED dataset. If you need to use it for SERV-CT dataset you need to further fine tune the usage, please see the paper for details. Or you can directly use the [weights](https://drive.google.com/drive/folders/19MaXDQ8mBM-g9Vtrh0Ka5XbJSJpYxQtG?usp=sharing) we obtained for the SERV-CT dataset.

**Comment by Tobyzai** [comment]
Thanks for your reply.

Using the model weights you provided for the servct dataset, the results are still poor.

The text mentions to resize the image size to (1280, 1024), is this necessary?

If so, how exactly is this handled?

Is the evaluation done on the original resolution (720, 576) or on the (1280, 1024) resolution?

Does the disparity map need to be multiplied by the appropriate scaling factor?

Looking forward to your reply.

**Comment by Tobyzai** [comment]
> Thanks for your reply.
> 
> Using the model weights you provided for the servct dataset, the results are still poor.
> 
> The text mentions to resize the image size to (1280, 1024), is this necessary?
> 
> If so, how exactly is this handled?
> 
> Is the evaluation done on the original resolution (720, 576) or on the (1280, 1024) resolution?
> 
> Does the disparity map need to be multiplied by the appropriate scaling factor?
> 
> Looking forward to your reply.

Dear author, do you have any idea about these?

**Comment by wurenkai** [comment]
We apologize for not being able to get back to you in a timely manner, as we have been meeting another project deadline recently. We think that different image sizes definitely have an impact on the parallax calculation for stereo matching. If you have used the weights we provided for inference, we recommend that you keep the same image size.

**Comment by Tobyzai** [comment]
Thanks, i will try.
